.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           mill -i __.publishArtifacts
           mill -i io.kipp.mill.ci.release.ReleaseModule/publishAll
         env:
-          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_PASSPHRASE: ""
```

Comment:
**thought:** the new key doesn't have a passphrase (generated in-browser on https://typelevel.org/sbt-typelevel/secrets.html)